index.html

```diff
@@ -4502,6 +4502,15 @@ <h3>Verification</h3>
             </ol>
           </li>
           <li>
+Ensure that the <var>result</var>.<var>mediaType</var> value is either
+`application/vc+ld+json` or `application/vp+ld+json` and does not contain any
+media type parameters, as defined in [[RFC6838]]. If the checks fail, set
```

Comment:
What about protocols that need to signal profiles of this specification?

What guidance do we provide for profiling?

Will it be ok to add `profile` in VCDM v3?

Reply:
You requested [that parameters are banned in this comment](https://github.com/w3c/vc-data-model/issues/1363#issuecomment-1839454755). I was implementing what you requested.

Data model "profiles" are signaled via `@context`.

We could add `profile` back in v3 by changing the algorithm and switching off of the `@context` version.

Exactly what sort of profiles do you foresee?

Reply:
I think it would be preferable to serve profiled data, without the need to sniff the context.

a profile parameter should be registered for `application/vc+ld+json` and when present, the verification algorithm should reject verification calls for media types it does not understand.

Reply:
Are we discussing V3 issues? Isn't this a bit premature when V2 is not in CR yet? 

I must admit I am also lost what we want and what we do not want.

I also note that if we are talking about a new feature that has not been discussed before, that should automatically be considered as a V3 issues. We have been in feature freeze for V2 for quite a while now...

Reply:
> I also note that if we are talking about a new feature that has not been discussed before, that should automatically be considered as a V3 issue. We have been in feature freeze for V2 for quite a while now...

Agreed.

Reply:
To be clear, my original request was in relation to the media type registration request.

When requesting new media types, you need to consider if parameters are allowed.

Given that `+ld+json` implies `profile`, my request was to acknowledge this explicit.

If we are saying `profile` is forbidden for `vc+ld+json`, in the registration, the text in this PR is good to be merged as is.